pages/common/sc_warts2pcap.md

```diff
@@ -0,0 +1,13 @@
+# sc_warts2pcap
+
+> Write packets included in warts object to a pcap file.
+> This is only possible for tbit, sting and sniff.
+> More information: <https://www.caida.org/catalog/software/scamper/>.
+
+- Convert the data from several warts files into one PCAP file:
+
+`sc_warts2pcap -o {{path/to/output.pcap}} {{path/to/file1.warts path/to/file2.warts ...}}`
+
+- Convert the data from a warts file into a PCAP file and sort the packets by timestamp:
```

Comment:
```suggestion
> Write packets included in `warts` object to a PCAP file.
> This is only possible for tbit, sting and sniff.
> More information: <https://www.caida.org/catalog/software/scamper/>.

- Convert the data from several `warts` files into one PCAP file:

`sc_warts2pcap -o {{path/to/output.pcap}} {{path/to/file1.warts path/to/file2.warts ...}}`

- Convert the data from a `warts` file into a PCAP file and sort the packets by timestamp:
```

Reply:
But you could also write `pcap` here, as with pcapfix, couldn't you?

Reply:
> But you could also write `pcap` here, as with pcapfix, couldn't you?

For file extensions and protocols we capitalize the terms instead of enclosing them inside backticks.

Reply:
But warts is also a file extension.

Reply:
> But warts is also a file extension.

Does it have a capitalized file format name? If so we could update to it instead.